### ISSUE TITLE
Support .NET client

### ISSUE THREAD

**Issue by Socolin** [vercel/cosmosdb-server#54]
Hello,

I started the server that way:
```
$ node lib/cli.js -p 8081
Ready to accept connections at [::]:8081
```

Then I installed the certificates so connection does not fail due to ssl error (If you are interesting I could write a PR for that, (the current `cert.pem` does not seems valid)

Then, when I try to connect to Cosmos DB using dotnet sdk, it's stuck

```
private const string ConnectionString = "AccountEndpoint=https://localhost:8081/;AccountKey=C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw==";
var cosmosClient = new CosmosClientBuilder(ConnectionString)
	.Build();
```

So I ran it in with the debugger and notice those messages spamming in the debug console I can see it try to connect to another port 

```
DocDBTrace Information: 0 : Marking endpoint https://localhost:3000/ unavailable for read
DocDBTrace Information: 0 : Current WriteEndpoints = (https://localhost:3000/) ReadEndpoints = (https://localhost:3000/)
DocDBTrace Information: 0 : Endpoint https://localhost:3000/ unavailable for Read added/updated to unavailableEndpoints with timestamp 12/20/2020 19:29:21
DocDBTrace Information: 0 : Resolving Master service address, forceMasterRefresh: False, currentMaster: 
```

This seems to be an hardcoded value from https://github.com/vercel/cosmosdb-server/blob/master/src/handler/read-meta.ts#L8

**Comment by nkzawa** [comment]
I think this is fixed by https://github.com/vercel/cosmosdb-server/issues/55 (released by the version `0.10.0`). Could you test it works for you?

**Comment by Socolin** [comment]
> I think this is fixed by #55 which removed hardcoded ports (released by the version `0.10.0`). Could you test it works for you?

Yeah it's fixed, now the problem seems to be the missing /addresses endpoint https://github.com/vercel/cosmosdb-server/issues/29
